### PR TITLE
[#2499] Fix projects notifications

### PR DESCRIPTION
**Pull request by danielkryska**

### CODE REVIEW

**Comment by Marcelinna** [comment]
If you change all notifications for consistency, change all of them:
"Category removed" -> "Category removed successfully"
"Platform destroyed" -> "Platform removed successfully"
"Provider has been removed" -> "Provider removed successfully"
"New scientific_domain created successfully" -> "New scientific domain created successfully"
"Scientific Domain removed" -> "Scientific Domain removed successfully " 
"Service destroyed" -> "Resource removed successfully"
"#{@Vocabulary type} removed" -> "#{@Vocabulary type} removed successfully"
"New offer has been created" -> "New offer created successfully"
and maybe other

**Review by jswk — CHANGES_REQUESTED** [review]
Have a look at several leftover "destoyed"s:
image

Just correct the commit message to conform to the repo's conventions and it seems good to go to me (with tests possing).

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -45,15 +45,15 @@ def edit; end
...
     end
   end
 
   def destroy
     if Project::Destroy.new(@project).call
-      redirect_to projects_path, notice: "Project destroyed"
+      redirect_to projects_path, notice: "Project deleted successfully"
     else
       redirect_to project_path(@project), alert: "Unable to remove project"
     end
```
Comment (Marcelinna) [inline app/controllers/projects_controller.rb:59]: Please change to Project removed successfully, we use word "removed" for other notification

Comment (jswk) [reply app/controllers/projects_controller.rb:59]: There's also "Profile information deleted successfully" in profiles_controller.rb if we want to unify this.

File: app/views/services/_paginate_simple.haml [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
       %li.page-item{ class: ("active" if per_page.blank? || per_page == "10") }
-        = link_to "10", params.merge(per_page: 10, page: 1).permit!, class: "page-link"
+        = link_to "10", params.merge(per_page: 10, page: 1), class: "page-link"
       %li.page-item{ class: ("active" if per_page == "20") }
-        = link_to "20", params.merge(per_page: 20, page: 1).permit!, class: "page-link"
+        = link_to "20", params.merge(per_page: 20, page: 1), class: "page-link"
       - if services.total_entries > 20
         %li.page-item{ class: ("active" if per_page == "30") }
-          = link_to "30", params.merge(per_page: 30, page: 1).permit!, class: "page-link"
+          = link_to "30", params.merge(per_page: 30, page: 1), class: "page-link"
```
Comment (jswk) [inline app/views/services/_paginate_simple.haml:11]: Why remove the `.permit!`?

**Review by jswk — CHANGES_REQUESTED** [review]
There's also still the phrase `deleted successfully` in the notices instead of `removed successfully`. Please correct these two remaining instances.

File: app/views/services/_paginate_simple.haml (hunk 2 above)
Comment (danielkryska) [reply app/views/services/_paginate_simple.haml:11]: `brakeman` throws errors in styles about it. After the removal of the `.permit!` flag page resources number is still working fine.

**Comment by danielkryska** [comment]
> There's also still the phrase `deleted successfully` in the notices instead of `removed successfully`. Please correct these two remaining instances.

done

**Review by Marcelinna — APPROVED** [review]

File: app/views/services/_paginate_simple.haml (hunk 2 above)
Comment (jswk) [reply app/views/services/_paginate_simple.haml:11]: How does one reproduce these errors? I run `brakeman` and don't get any errors. I can see these lines being ignored in `brakeman.ignore`.

These `.permit!`s seem indeed not necessary here though. I would extract these changes to a separate commit and also update the `brakeman.ignore` in that commit then by running an interactive `brakeman -I`.

**Review by jswk — CHANGES_REQUESTED** [review]
Extract the unrelated `.permit!` changes to a separate commit and update `brakeman.ignore` appropriately.

**Review by jswk — APPROVED** [review]

**Pull request merged**